### PR TITLE
Issue #207 Adding ROADMAP.md

### PR DESCRIPTION
**Pull request by hferentschik**
Addressing issue #207 - Adding a high level roadmap

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+**Run OpenShift locally**
+
+Minishift's goal is to become the default choice for running a single-node OpenShift cluster on
+your local host. Either for evaluation purposes or for ongoing development.
+
+Due to the fact that Minishift builds upon OpenShift's [cluster up](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md),
+Minishift's focus lies on providing more value to the basic _cluster up_ provisioning.
+This includes virtual machine creation and management, making it easier to configure
+OpenShift in preceicly the way you want and providing useful shortcut commands for making your
```
Comment (budhrg) [inline ROADMAP.md]: precisely

**Comment by budhrg** [comment]
Overall, LGTM :+1:  with one typo, rest I trust @thatdocslady and @Preeticp comments :smile:

File: ROADMAP.md (hunk 1 above)
Comment (hferentschik) [inline ROADMAP.md]: Fixed, thanks

**Comment by hferentschik** [comment]
@LalatenduMohanty WDYT?

**Comment by praveenkumar** [comment]
LGTM 👍

**Comment by LalatenduMohanty** [comment]
@hferentschik LGTM. Lets merge it and then if required will update this later.

**Comment by hferentschik** [comment]
>  LGTM. Lets merge it and then if required will update this later.

Awesome, thanks

**Pull request merged**